### PR TITLE
Fastlane releases

### PR DESCRIPTION
**Pull request by marcosgriselli**
This PR adds support release automation using [Fastlane](https://fastlane.tools). It will update the podspec version, Info.plist and tests Info.plist with `bump_type` of this release. We could even analyze the possibility of adding it to the PR merging flow so Travis can release when we merge to `master`.

### Usage
From CodableFirebase directory run: 
`bundle exec fastlane [patch | minor | major]`

### CODE REVIEW

File: fastlane/Appfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
```
Comment (alickbass) [inline fastlane/Appfile]: Do we even need this file? I think we can remove it completely

Comment (marcosgriselli) [reply fastlane/Appfile]: Removed.

**Comment by alickbass** [comment]
Thanks a lot for this PR! I really appreciate the support from the community! About travis hooks, I think there will be a problem with pushing to git from Travis because it checks out from detached commit and not just the branch... but we can investigate further

**Comment by marcosgriselli** [comment]
Sounds good. Using this approach is good enough though, creating a new release with just one command is pretty convenient, I have all my libraries configured like this.

**Pull request merged**